bindings/hyperdrivepy/src/hyperdrive_state_methods.rs

```diff
@@ -169,6 +170,18 @@ impl HyperdriveState {
         Ok(result)
     }
 
+    pub fn calculate_open_short_deltas(&self, bond_amount: &str) -> PyResult<String> {
+        let bond_amount_fp = FixedPoint::from(U256::from_dec_str(bond_amount).map_err(|_| {
+            PyErr::new::<PyValueError, _>("Failed to convert bond_amount string to U256")
+        })?);
+        let result_fp = self
+            .state
+            .calculate_open_short_deltas(bond_amount_fp)
+            .unwrap();
```

Comment:
don't we want to handle the error here with a `map_err()`?

Reply:
yeah I like that a lot better. The best example is in `calculate_targeted_long_with_budget`, where we do:

```
            .map_err(|err| {
                PyErr::new::<PyValueError, _>(format!(
                    "Calculate_targeted_long_with_budget returned the error: {:?}",
                    err
                ))
            })?;
```

But that's the only place we do it. Again I think it'd be better to fix this everyone in a single PR. I'll make note & do it.

https://github.com/delvtech/hyperdrive-rs/issues/64